tests/testthat/test-fixed_regex_linter.R

```diff
@@ -300,51 +300,59 @@ robust_non_printable_unicode <- function() {
 }
 
 # styler: off
-patrick::with_parameters_test_that("fixed replacements are correct", {
-  skip_if(
```

Comment:
Maybe `skip_if(regex_expr %in% skip_cases, ...)` to still keep the skips visible while also making the code easier to read?

Reply:
see https://github.com/google/patrick/issues/19, I wanted to avoid running skip_if() every iteration. I guess we can restore that approach in the meantime for readability.

Reply:
Yes. With the conditional factored out, this will be more readable than subsetting I think.